Microsoft.Activities.Build/Microsoft/Activities/Build/BeforeInitializeComponentExtension.cs

```diff
@@ -0,0 +1,242 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
```

Comment:
You seem to be missing the files directly under Microsoft.Activities.Build:

Microsoft.Activities.Build.csproj
Microsoft.Activities.Build.targets
Settings.StyleCop
SR.resx

Reply:
Oops. Microsoft.Activiites.Build.Targets should be Microsoft.WorkflowBuildExtensions.targets

Reply:
They are now added to the PR